.flake8

```diff
@@ -4,3 +4,4 @@ exclude =
     __pycache__,
     build
 max-complexity = 10
+max-line-length= 105
```

Comment:
You may need to change this for the isort and black config settings in the pyproject.toml file.

Reply:
I guess I would have to change those instead. If I don't increase the `max-line-length` flake gives me the length warning due to the documentation. It could be easily solved if I just go to a new line in the docs but since I am giving an example of an input dictionary, for some reason Sphinx gives me also a warning. I'm not an expert in Sphinx, so any recommendation would be highly appreciated

Reply:
You should be able to configure flake8 to ignore certain files/directories